services/movie_session.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+
+from db.models import MovieSession
+from django.db.models import QuerySet
+
+
+def create_movie_session(
+        movie_show_time: datetime, movie_id: int, cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(session_date: str = None, ) -> QuerySet:
```

Comment:
The return type of the `get_movies_sessions` function is declared as `QuerySet`, but it should be `QuerySet[MovieSession]` to specify that the queryset contains `MovieSession` objects. This aligns with the checklist point: 'the `.get()` method returns a single object, not a `QuerySet`.'